playground/src/workbench/prql-syntax.js

```diff
@@ -1,26 +1,26 @@
+const TRANSFORMS = [
+  "from",
+  "select",
+  "derive",
+  "filter",
+  "take",
+  "sort",
+  "join",
+  "aggregate",
+  "group",
+  "window",
+  "concat",
+  "union",
+];
+const BUILTIN_FUNCTIONS = ["switch"]; // "in", "as"
+const KEYWORDS = ["func", "table", "prql"];
+const LITERALS = ["and", "or", "not", "null", "true", "false"];
```

Comment:
Technically `and` & `or` are operators rather than Literals

Also `not` is actually not a thing in PRQL, I realize it was there before

If you want to make those changes then great, otherwise fine to merge as-is

Reply:
Good points - updated.

The literal thing was only because highlight-prql.js passes them to that key, but there's no need to copy the hljs fudge over.